### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by AvidCoder101**
I have made a CONTRIBUTING.md file. Please send any feedback if needed.

### CODE REVIEW

**Comment by MRslimcoder** [comment]
@AvidCoder101 don't merge this PR, will add changes within this PR by myself.

**Comment by AvidCoder101** [comment]
I still needed someone to check it for me first anyway, in case I needed something changing.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+### 2. Clone the repository :
+
+![image of cloning](https://user-images.githubusercontent.com/70807684/126664891-61fdf492-976d-49f1-a2ec-5ed23eeca06e.png)
+
+
+
+#### **ON YOUR FORKED REPOSITORY**, copy the link by pressing the clipboard and on your local git, copy the link in. If you are unable, use the link below and replace `username` with your GithHub username :
+
+                                                   https://github.com/username/List-of-Learning-Resources.git
```
Comment (MRslimcoder) [inline CONTRIBUTING.md:18]: This shouldn't be in the center.

**Comment by MRslimcoder** [comment]
@AvidCoder101 I push all changes within your PR.

**Review by MRslimcoder — APPROVED** [review]
LGTM

**Comment by MRslimcoder** [comment]
Capture6

@AvidCoder101 can you please remove your last commit it is not looking good.

**Comment by AvidCoder101** [comment]
Sorry, just saw that you had changed it. I fixed it back.

**Comment by MRslimcoder** [comment]
@AvidCoder101 remove your two last commits and then git push.

**Comment by AvidCoder101** [comment]
@MRslimcoder I've just changed it back. Is that ok?

**Comment by MRslimcoder** [comment]
LGTM

**Pull request merged**